.github/workflows/build-python-pdoc.yml

```diff
@@ -0,0 +1,51 @@
+name: DiskANN Build PDoc Documentation
+on: [workflow_call]
+jobs:
+  build-reference-documentation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: "Run Reference Documentation Generation"
+        run: |
+          pip install diskannpy pdoc pipdeptree
+          echo "documentation" > dependencies_documentation.txt
+          pipdeptree >> dependencies_documentation.txt
+          pdoc -o docs/python/html diskannpy
+      - name: Create version environment variable
+        run: |
+          echo "DISKANN_VERSION=$(python <<EOF
+          from importlib.metadata import version
+          v = version('diskannpy')
+          print(v)
+          EOF
+          )" >> $GITHUB_ENV
```

Comment:
For some reason "python setup.py --version" did not work on Linux.  It did work on Windows though.  This code will run in Bash.

Reply:
In the python-release workflow, it's only triggered on a new release being published.  I bet we can get the version/tag from that, which would have the version number.

Alternatively, since we'll probably have the current release installed in the job due installing it from the wheel, we can also probably do the following: `python -c "import diskannpy; print(diskannpy._diskannpy.__version__);"` and capture it there